### PR TITLE
Clarify use of booleans and conditionals for dynamic options

### PR DESCRIPTION
**Pull request by lparsons**

### CODE REVIEW

**Review by hexylena — APPROVED** [review]
:+1:

File: docs/best_practices/tool_xml.rst [hunk 1]
```diff
@@ -106,15 +106,16 @@ section.
...
+below.
 
 Dynamic Options
 ---------------
 
-Certain options such as "Advanced Options" should be a select box and
-not a boolean. A checkbox is not expected to change the content of the
-mask from a user point of view.
+Options that are conditionally hidden (such as "Advanced Options") should be a
+use a ``select`` input type and not a boolean. The user should not expect a
```
Comment (bgruening) [inline docs/best_practices/tool_xml.rst]: `should be a use a` -> `should use a`

Comment (lparsons) [reply docs/best_practices/tool_xml.rst]: Thanks @bgruening, fixed.

**Comment by martenson** [comment]
Thanks @lparsons , sorry for the merge delay!

**Pull request merged**